### PR TITLE
Add `cargo-tarpaulin` support

### PR DESCRIPTION
**Pull request by orhun**
See #42

### CODE REVIEW

File: tools/codegen/base/cargo-tarpaulin.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "tag_prefix": "",
+  "asset_name": "${package}-${rust_target}.tar.gz",
+  "version_range": "> 0.22.0",
+  "platform": {
+    "x86_64_linux_gnu": {},
+    "x86_64_linux_musl": {},
+    "aarch64_linux_gnu": {},
+    "aarch64_linux_musl": {},
+    "x86_64_macos": {
+      "asset_name": "${package}-universal-apple-darwin.tar.gz"
```
Comment (taiki-e) [inline tools/codegen/base/cargo-tarpaulin.json:12]: Why not use x86_64 binaries?

**Review by taiki-e — COMMENTED** [review]
Thanks!

File: tools/codegen/base/cargo-tarpaulin.json (hunk 1 above)
Comment (orhun) [reply tools/codegen/base/cargo-tarpaulin.json:12]: Not sure, took that from here:

https://github.com/taiki-e/install-action/blob/f30d182e9eb6ec2e7e967394d0db4b42f5984431/tools/codegen/base/cargo-nextest.json#L8-L10

I can change it if you want 🐻

Comment (orhun) [reply tools/codegen/base/cargo-tarpaulin.json]: PR is updated!

**Comment by taiki-e** [comment]
Published in 2.4.0.

**Pull request merged**